styles.css

```diff
@@ -289,11 +280,85 @@ textarea:focus {
 /* --------------- m o b i l e ----------------------- */
 
 @media only screen and (max-width: 767px) {
+  header {
+    margin-left: 5%;
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    width: 100%;
+    padding-top: 5%;
+  }
+
   .logo h3 {
-    font-weight: 600px;
+    font-weight: 600;
     font-size: 20px;
   }
 
+  /* dropdown menu start */
+  .container {
+    display: inline-block;
+    cursor: pointer;
+    align-items: center;
+  }
+
+  .bar1,
+  .bar2,
+  .bar3 {
+    width: 35px;
+    height: 5px;
+    background-color: #333;
+    margin: 6px 0;
+    transition: 0.4s;
+  }
+
+  .change .bar1 {
+    transform: translate(0, 11px) rotate(-45deg);
+    background-color: #67798e;
+  }
+
+  .change .bar2 {
+    opacity: 0;
+  }
+
+  .change .bar3 {
+    transform: translate(0, -11px) rotate(45deg);
+    background-color: #67798e;
+  }
+
+  .accordion {
+    color: #444;
+    cursor: pointer;
+    width: 30%;
+    border: none;
+    outline: none;
+    font-size: 15px;
+    transition: 0.4s;
+  }
```

Comment:
- [ ] Kindly reduce the size of the closing icon so that it looks identical to that of the Figma template that you chose. The one you have is too big. Also, the `font-weight` of the closing icon is too thick. It would be nice to reduce it. Kindly implement this change. 👍 🆒 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/223191509-d7c86745-689a-4ec5-931d-53f0db0e316b.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/223191604-dbd335b3-28ba-4023-801d-49fb1ad4ce27.png)

Reply:
Adjusted accordignly